config/gcos.words

```diff
@@ -0,0 +1,478 @@
+reserved:	BINARY-SHORT		# COBOL 2002
+reserved:	BIT			# COBOL 2002
+reserved:	BLANK
+reserved:	BLOCK
+reserved:	BOOLEAN			# COBOL 2002
+reserved:	BOTTOM
+reserved:	BSN
+reserved:	BY
+reserved:	CALL
+reserved:	CANCEL
+reserved:	CD
+reserved:	CF
+reserved:	CH
+reserved:	CHARACTER
+reserved:	CHARACTERS
+reserved:	CLASS
+reserved:	CLOCK-UNITS
+reserved:	CLOSE
+reserved:	COBOL
+reserved:	CODE
+reserved:	CODE-SET
+reserved:	COLLATING
+reserved:	COLUMN
+reserved:	COMMA
+reserved:	COMMON
+reserved:	COMMUNICATION
+reserved:	COMP=COMP-3
+reserved:	COMP-1=BINARY-SHORT
+reserved:	COMP-2=BINARY-LONG
+reserved:	COMP-3
+reserved:	COMP-5=PACKED-DECIMAL
+reserved:	COMP-8=PACKED-DECIMAL
+reserved:	COMP-9=FLOAT-SHORT
+reserved:	COMP-10=FLOAT-LONG
+reserved:	COMP-15=FLOAT-LONG
+reserved:	COMPUTATIONAL=COMP-3
+reserved:	COMPUTATIONAL-1=BINARY-SHORT
+reserved:	COMPUTATIONAL-2=BINARY-LONG
+reserved:	COMPUTATIONAL-3
+reserved:	COMPUTATIONAL-5=PACKED-DECIMAL
+reserved:	COMPUTATIONAL-8=PACKED-DECIMAL
+reserved:	COMPUTATIONAL-9=FLOAT-SHORT
+reserved:	COMPUTATIONAL-10=FLOAT-LONG
+reserved:	COMPUTATIONAL-15=FLOAT-LONG
+reserved:	COMPUTE
+reserved:	CONFIGURATION
+reserved:	CONSTANT		# COBOL 2002 != GCOS (unsupported)
+reserved:	CONTAINS
+reserved:	CONTENT
+reserved:	CONTINUE
+reserved:	CONTROL
+reserved:	CONTROLS
+reserved:	CONVERSION		# not supported yet
+reserved:	CONVERTING
+reserved:	COPY
+reserved:	CORR
+reserved:	CORRESPONDING
+reserved:	COUNT
+reserved:	CURRENCY
+reserved:	DATA
+reserved:	DATE
+reserved:	DATE-COMPILED
+reserved:	DATE-WRITTEN
+reserved:	DAY
+reserved:	DAY-OF-WEEK
+reserved:	DE
+reserved:	DEBUGGING
+reserved:	DECIMAL-POINT
+reserved:	DECLARATIVES
+reserved:	DEFAULT			# COBOL 2002 != GCOS (unsupported)
+reserved:	DELETE
+reserved:	DELIMITED
+reserved:	DELIMITER
+reserved:	DEPENDING
+reserved:	DESCENDING
+reserved:	DESTINATION
+reserved:	DETAIL
+reserved:	DISABLE
+reserved:	DISPLAY
+reserved:	DIVIDE
+reserved:	DIVISION
+reserved:	DOWN
+reserved:	DUPLICATES
+reserved:	DYNAMIC
+reserved:	EBCDIC
+reserved:	EGI
+reserved:	ELSE
+reserved:	EMI
+reserved:	ENABLE
+reserved:	END
+reserved:	END-ADD
+reserved:	END-CALL
+reserved:	END-COMPUTE
+reserved:	END-DELETE
+reserved:	END-DIVIDE
+reserved:	END-EVALUATE
+reserved:	END-IF
+reserved:	END-MULTIPLY
+reserved:	END-OF-PAGE
+reserved:	END-PERFORM
+reserved:	END-READ
+reserved:	END-RECEIVE
+reserved:	END-RETURN
+reserved:	END-REWRITE
+reserved:	END-SEARCH
+reserved:	END-START
+reserved:	END-STRING
+reserved:	END-SUBTRACT
+reserved:	END-UNSTRING
+reserved:	END-WRITE
+reserved:	ENTER
+reserved:	ENVIRONMENT
+reserved:	EOP
+reserved:	EQUAL
+reserved:	EQUALS=EQUAL
+reserved:	ERROR
+reserved:	ESI
+reserved:	EVALUATE
+reserved:	EVERY
+reserved:	EXCEEDS=GREATER
+reserved:	EXCEPTION
+reserved:	EXIT
+reserved:	EXTEND
+reserved:	EXTERNAL
+reserved:	FALSE
+reserved:	FD
+reserved:	FILE
+reserved:	FILE-CONTROL
+reserved:	FILLER
+reserved:	FINAL
+reserved:	FIRST
+reserved:	FLOAT-LONG		# COBOL 2002
```

Comment:
If this isn't supported in GCOS, then please drop it here.
If it is drop the COBOL 2002 comment, also in the other places.

Reply:
Got rid of the coments.